src/catalog/handlers/uadetrac_table_handler.py

```diff
@@ -0,0 +1,57 @@
+"""
+Defines UADETRAC TableHandler.
+Provides the methods to infer stats of a Table.
+"""
+import pandas as pd
+
+from src.catalog.sqlite_connection import SqliteConnection
+from src import constants
+from src.catalog.handlers.base_table_handler import BaseTableHandler
+
+class VideoFrameMap:
+    def __init__(self, args):
+        self.id = args['id']
+        self.video_id = args['video_id']
+
+
+class UaDetracTableHandler(BaseTableHandler):
+    """
+
+    """
+
+    def __init__(self, conn: SqliteConnection):
+        self.dataset_name = constants.UADETRAC
+        BaseTableHandler.__init__(conn)
+        self.table_name = self.dataset_name + "_mapping"
+        self.conn = conn
+
+    def __str__(self):
+        return 'Frame(table_name=' + self.table_name + ',id=' + str(self.id) \
+               + ',video_id=' + str(self.video_id) + ')'
+
+    def create_uadetrac_table(self, conn):
```

Comment:
This could have been more abstracted out. I mean if we later have to change the schema for some reason, it will be pain to go in the file and change the file each time. And by this design, we will have to create a separate handler for each dataset.
An architecture where we save the schema of all tables in a central repository and a common utility to perform create, add_mapping on such schema seems a better design here. What are your views?